include/elements/2d/quadrilateral_bspline_element.tcc

```diff
@@ -13,6 +14,20 @@ void mpm::QuadrilateralBSplineElement<Tdim, Tpolynomial>::
   //! Uniform spacing length in 2D
   this->spacing_length_ =
       std::abs(nodal_coordinates(1, 0) - nodal_coordinates(0, 0));
+
+  //! Identify if element is at boundary to see if kernel correction is
```

Comment:
Just to see if we are at the same point. These lines below check if a specific cell is at the boundary. And then just activate the kernel correction at those cells. I'm right?

Reply:
Correct! This `bool` is activated at `2.0 * spacing_length_` which will be further checked by the function `kernel_correction_region()` for the exact `1.5 * spacing_length_` distance.